docs/ldml/tr35.md

```diff
@@ -1312,14 +1315,20 @@ Examples:
 <tr><td colspan="3">…</td></tr>
 </tbody></table>
 
+ _However, if users typically expect a their language in non-default script to contain an admixture of another language, then the -t- and hybrid subtags may be omitted. Thus if Hindi written in Latin script normally contains an admixture of English text, ‘hi-Latn’ can be used instead of ‘hi-Latn-t-en-h0-hybrid’._
```

Comment:
Same as earlier comment about it may be helpful to talk about borrowing.

Reply:
Hi, just fixed. Can you rereview?

Reply:
do you need to fix line 1319 too?